docs/ups_configuration.rst

```diff
@@ -0,0 +1,24 @@
+UPS Carrier Configuration
+=========================
+
+Before any shipment can be sent to the customer address, proper details about
+the carriers handling the parcels and shipments need to be attached with the 
+purchase order.
+
+The aim of this documentation is to help the user understand how to configure
+UPS Carrier for addition to purchase orders of customers. 
+
+1. Click on ``Inventory & Stock`` >> ``Configuration``. 
+2. Double click on ``UPS Configuration``. 
+
+In the details window, double click on UPS Configuration.
+
+Fill in the details associated with UPS Account and then
+click on ``Save this record``.
+
+.. figure:: images/ups_configuration.png
+
+UPS Carrier configuration has been successfully done.
+
+
```

Comment:
this section should also explain how to obtain the API access key and related information. Shopify explains this https://docs.shopify.com/manual/settings/shipping/carrier-calculated-rates#xml-access-key